uw-frame-components/portal/main/partials/footer.html

```diff
@@ -5,7 +5,7 @@
         <span>© <p style="display:inline">{{date | date:'yyyy'}}</p>, Board of Regents of the <a href="http://www.wisconsin.edu/" target="_blank">University of Wisconsin System</a></span>
       </div>
       <div class='underline'>
-        <span ng-repeat="link in FOOTER_URLS">&nbsp;<a ng-href='{{link.url}}' target='{{link.target}}' ng-click="pushGAEvent('footer-link',link.title,link.url + ' ' + link.target)">{{link.title}}</a></span>
+        <span ng-repeat="link in portal.theme.FOOTER_LINKS">&nbsp;<a ng-href='{{link.url}}' target='{{link.target}}' ng-click="pushGAEvent('footer-link',link.title,link.url + ' ' + link.target)">{{link.title}}</a></span>
```

Comment:
Instead of replacing the `FOOTER_URLS` lets just append to it. Make the `portal.theme.footerLinks` go first, then the `FOOTER_URLS`. So two ng-repeat blocks. You will want to tweak the `app-config.js` so there is no help link.